packages/medusa/src/utils/flag-router.ts

```diff
@@ -14,4 +14,10 @@ export class FlagRouter implements IFlagRouter {
   public setFlag(key: string, value = true): void {
     this.flags[key] = value
   }
+
+  public getFlags(): string[] {
+    return Object.keys(this.flags || {}).filter((flag) =>
+      this.isFeatureEnabled(flag)
+    )
+  }
```

Comment:
suggestion: maybe `listFlags`?

Reply:
Would also include all flags - i.e. not just the ones that are enabled

Reply:
Also, let's make the response like this - this way we can add additional details in the future if necessary:
```
[
  { key: [..], value: [...] },
  ...
]
```

Imagine we wanted to add `scope` for example `["store"]`, `["admin", "store"]`.